Sukanta_Roy/progress.md

```diff
@@ -3,78 +3,66 @@
 **Project: Questions+**
 
 Overall Direction:
-- 
-- 
-- 
-
+- Rubric questions: allow weightages for sub questions
+- MCQ questions: support attaching weights to options
+- Calculate more statistics for MCQ questions (Total, average for each student)
+- MSQ questions: support attaching weights to options
+- Calculate more statistics for MSQ questions
+- Ensure the downloaded data contains all data shown in the page.
+- Add a new question template
 ---
 
 Week 1 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Make UI changes for [Rubric questions: allow weightages for sub questions](https://github.com/TEAMMATES/teammates/issues/7224)
```

Comment:
I don't think separate the work of writing test is a good option (Remember TDD :) ).
Instead, u can first write backend, frontend and then documentation. The test should be **along** with the changes.
You can have a PR pending (so we can also make sure you are in the right path) and gradually push your commits.
- Adds backend changes for rubric weightages
- Adds test cases for rubric weightage in backend
- ......

Reply:
@xpdavid Actually, when you said the first few weeks of the plan should be detailed, I thought that I should divide the work into smaller segments so that tracking the progress through my plan would be easy. So, in that dividing process I divided the backend part into logic and tests. :sweat_smile: 

Changing this now.